### PR TITLE
Tweaked language to allow for multiple official versions of the scrambling software.

### PR DESCRIPTION
**Pull request by jfly**
The motivation for this change is that new TNoodle versions with new features always get delayed for months because we don't like to mark new releases of TNoodle as official if they don't change the way scrambles are generated. However, we wrote TNoodle and the WCA API with this problem in mind, and as far as all our software is concerend, it is possible to have multiple official versions of TNoodle.

The current language still implies that there is only one official scrambling software program... maybe in some future we'll have multiple offical scrambling softwares, but I think it would be best to address that when it comes.

### CODE REVIEW

**Comment by lgarron** [comment]
> The motivation for this change is that new TNoodle versions with new features always get delayed for months because we don't like to release new versions of TNoodle that don't change the way scrambles are generated.

If by "don't like" you mean "it's literally not allowed for Delegates to use those". ;-)

I think it's perfectly reasonable to allow multiple official scramble programs, but all of them must comply with the Regulations.

File: wca-regulations.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 4d3) Square-1 is scrambled starting with the darker colour on front (out of the 2 possible scrambling orientations).
     - 4d4) Clock is scrambled starting with either side in front and 12 o'clock pointing up.
     - 4d5) Skewb is scrambled starting with the white face (if not possible, then the lightest face) on top and the green face (if not possible, then the darkest adjacent face) on the front-left.
-- 4f) Competition scramble sequences must be generated using the current official version of the official WCA scramble program (available [via the WCA website](https://www.worldcubeassociation.org/regulations/scrambles/)).
+- 4f) Competition scramble sequences must be generated using one of the current official versions of the official WCA scramble program (available [via the WCA website](https://www.worldcubeassociation.org/regulations/scrambles/)).
```
Comment (lgarron) [inline wca-regulations.md]: I would simplify this to "a current official version"

Comment (jfly) [reply wca-regulations.md]: Done!

**Comment by jfly** [comment]
> If by "don't like" you mean "it's literally not allowed for Delegates to use those". ;-)

Sorry, I abused the vocabulary here. Let me rephrase: "we don't like to make new versions of TNoodle official if they don't change the way scrambles are generated"

> I think it's perfectly reasonable to allow multiple official scramble programs, but all of them must comply with the Regulations.

I also have no problem with multiple official scramble programs, I just could not think of a good rephrasing that allows for this, so I gave up. Do you have a suggestion?

**Comment by lgarron** [comment]
> I also have no problem with multiple official scramble programs, I just could not think of a good rephrasing that allows for this, so I gave up. Do you have a suggestion?

I don't think this needs further coverage. It seems obvious to me that Delegates should not be allowed to use scramble programs that don't follow the Regulations.

**Comment by lgarron** [comment]
Anyhow, I'm happy to merge this. @Laura-O, @Claster, @viroulep can I get a second approval from one of you to merge this?

**Comment by jfly** [comment]
> It seems obvious to me that Delegates should not be allowed to use scramble programs that don't follow the Regulations

I think that is clear, and is not the issue I'm trying to address. What is unclear is if there were multiple scramble programs that _do_ follow the regulations, would the wca regulations allow for declaring both of them official?

**Comment by lgarron** [comment]
> if there were multiple scramble programs that do follow the regulations, would the wca regulations allow for declaring both of them official?

Oh, you mean different programs (like TNoodle and Mark 2) rather than different versions of the "same" program?
I don't see a problem with writing "a current official version of an official WCA scramble program"

**Comment by Laura-O** [comment]
@lgarron: as already said in the mail thread: let's get this done. 👍

**Comment by jfly** [comment]
Darn it! I just made some relevant changes.

**Comment by jfly** [comment]
New PR incoming.

(edit: See #398)

**Comment by lgarron** [comment]
[Reverted. Let's continue on #398.]
